protocols/raft/src/main/java/io/atomix/protocols/raft/metrics/LeaderMetrics.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016-present Open Networking Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.atomix.protocols.raft.metrics;
+
+import io.prometheus.client.Histogram;
+
+public class LeaderMetrics {
+
+  private static final Histogram APPEND_LATENCY =
+      Histogram.build()
+          .namespace("atomix")
+          .name("append_entries_latency_ms")
+          .help("Latency to append an entry to a follower")
+          .labelNames("to", "partitionName")
+          .register();
+
+  private final String partition;
+
+  public LeaderMetrics(String partitionName) {
+    int partitionId;
+    try {
+      final String[] parts = partitionName.split("-");
+      partitionId = Integer.valueOf(parts[parts.length - 1]);
+    } catch (Exception e) {
+      partitionId = 0;
```

Comment:
Can we log this in case it happens?

Reply:
I was not sure if I should extract the partition id from the name because Raft can be used with out partitions also. If `partitionName` is not in the expected format, it usually means, it is a standalone raft. I can log it. But it is not really an exception case, right?

Reply:
Yes, not an exception, so no need to log it to error level, but it's useful information for us in case it ever happens. Debug or info is fine.